cschwitalla/WES_Analysis/functions_exome.R

```diff
@@ -0,0 +1,52 @@
+################################################################################
+###                                Purpose                                   ###
+################################################################################
+## Title: Functions for WES Analysis 
+## Author: Carolin Schwitalla
+## 
+## Description: This script contains all functions that are used for the WES 
+##              analysis for my master thesis 
+################################################################################
+###                                                                          ###
+################################################################################
+#Variant allel Frequency
+# ----- from Rike
+# Calculate Varaint allele frequency
+compute_vaf <- function(maf_obj){
+  genes_vaf = subsetMaf(maf = maf_obj, query = "t_depth > 0", 
+                        fields = c("t_depth","t_alt_count"), mafObj = FALSE)
+  #print(genes_vaf$t_alt_count)
+  genes_vaf$VAF <- as.numeric(genes_vaf$t_alt_count)/as.numeric(genes_vaf$t_depth)
+  genes_vaf <- genes_vaf[, mean(VAF), Hugo_Symbol]
+  colnames(genes_vaf)[2] = "VAF"
+  genes_vaf <- genes_vaf[which(genes_vaf$VAF > 0.02)]
+  return(genes_vaf)
+}
+
+
+
+
+patientcolor2 <- setNames(c("#543005", "#8c510a", "#A6691C", "#bf812d", "#CFA255", 
```

Comment:
See comment from the other file, if you use this so often make a file where you obtain this information.

OR since, this is the function script, define the vector here and remove it from the other script